blocks/text.js

```diff
@@ -689,3 +689,90 @@ Blockly.Blocks['text_prompt'] = {
   mutationToDom: Blockly.Blocks['text_prompt_ext'].mutationToDom,
   domToMutation: Blockly.Blocks['text_prompt_ext'].domToMutation
 };
+
+Blockly.Blocks['text_count'] = {
+  /**
+   * Block for counting how many times one string appears within another string.
+   * @this Blockly.Block
+   */
+  init: function() {
+    this.jsonInit({
+      'message0': Blockly.Msg.TEXT_COUNT_MESSAGE0,
+      'args0': [
+        {
+          'type': 'input_value',
+          'name': 'SUB',
+          'check': 'String',
+        },
+        {
+          'type': 'input_value',
+          'name': 'TEXT',
+          'check': 'String',
+        },
+      ],
+      'output': 'Number',
+      'inputsInline': true,
+      'colour': Blockly.Blocks.math.HUE,
+      'tooltip': Blockly.Msg.TEXT_COUNT_TOOLTIP,
+      'helpUrl': Blockly.Msg.TEXT_COUNT_HELPURL,
```

Comment:
Yup, that's fine. I just didn't realise this object literal was meant to be written as if it was inside a JSON string.